modules/gke-cluster/variables.tf

```diff
@@ -70,10 +70,25 @@ variable "logging_service" {
 }
 
 variable "dashboard_disabled" {
-  default = false
+  default     = false
   description = "Disable Kubernetes Dashboard"
 }
 
+variable "master_auth_username" {
+  default     = ""
+  description = "The username to use for HTTP basic authentication"
+}
+
+variable "master_auth_password" {
+  default     = ""
```

Comment:
What happens if `master_auth_(username|password)` is the default value, i.e. empty string? Might be worth a comment (perhaps in the 'description' field).

Reply:
Yeah good point, I'll add a comment - when empty, it actually disables the basic auth (that's what we want).